mail_activity_board/models/mail_activity.py

```diff
@@ -41,9 +37,7 @@ def open_origin(self):
         return response
 
     @api.model
-    def action_your_activities(self):
+    def action_activities_board(self):
         action = self.env.ref(
-            'mail_activity_board.open_boards_activities_form_tree').read()[0]
-        action_context = safe_eval(action['context'], {'uid': self.env.uid})
-        action['context'] = action_context
+            'mail_activity_board.open_boards_activities').read()[0]
```

Comment:
just do `self.env.ref(...).id`

Reply:
They do not return the same.
self.env.ref ('...'). read () [0] returns a dictionary with all the data of the action.
self.env.ref ('...'). id only returns the id (int) of the action.

I can not change it